lib/threads.cpp

```diff
@@ -520,7 +521,7 @@ int GLVisCommand::Execute()
          cout << "Command: keys: '" << key_commands << "'" << endl;
          // SendKeySequence(key_commands.c_str());
          CallKeySequence(key_commands.c_str());
-         MyExpose();
+         SendExposeEvent();
```

Comment:
There is a reason why here (and in many other places) we use `MyExpose` and not `SendExposeEvent` -- the first one is a synchronous update and the second is asynchronous. This is an important distinction, e.g. when taking screenshots via commands.